python/src/experimental_module.i

```diff
@@ -72,13 +72,14 @@
 
 /* Base/Algo */
 %include CubaIntegration.i
+%include SimplicialCubature.i
```

Comment:
this is part of the algo module

Reply:
??? I never put it in this place! A bad merge